docs/python/1.6/python-appapi-devguide-4.md

```diff
@@ -378,10 +378,68 @@ Sample output:
 {'min': '5', 'id': '6508', 'max': '25', 'timestamp': '1551729422809'}
 ~~~~~
 
+### Using streamsx.standard for reading and writing of files
+
+Another option to work with files in your streaming application is offered by the Python module [streamsx.standard](https://streamsxstandard.readthedocs.io/en/latest/generated/streamsx.standard.files.html#module-streamsx.standard.files).
+
+In the following sample, the [streamsx.standard.files.CSVReader](https://streamsxstandard.readthedocs.io/en/latest/generated/streamsx.standard.files.html#streamsx.standard.files.CSVReader) is used to read a file in the source callable. For this, we import the python module like below:
+
+~~~~~ python
+import streamsx.standard.files as files
+~~~~~
+
+The [streamsx.standard.files.CSVReader](https://streamsxstandard.readthedocs.io/en/latest/generated/streamsx.standard.files.html#streamsx.standard.files.CSVReader) accepts the file parameter either set as relative path to application directory or as absolute path.
+In this sample the file is added application bundle and the Streams application reads this file at runtime from the application directory.
```

Comment:
In this sample, the file is added to the application bundle ...
"'to the" is missing.